INTERFACE_SPECIFICATION.md

```diff
@@ -0,0 +1,272 @@
+## The Bitcoin Canister API
+
+The canister IDs of the Bitcoin canisters for Bitcoin mainnet and testnet:
+
+* `mainnet`:  [`ghsi2-tqaaa-aaaan-aaaca-cai`](https://dashboard.internetcomputer.org/canister/ghsi2-tqaaa-aaaan-aaaca-cai)
+* `testnet` (v4):  [`g4xu7-jiaaa-aaaan-aaaaq-cai`](https://dashboard.internetcomputer.org/canister/g4xu7-jiaaa-aaaan-aaaaq-cai)
+
+Information about Bitcoin and the IC Bitcoin integration can be found in the [Bitcoin developer guides](https://developer.bitcoin.org/devguide/) and the [Bitcoin integration documentation](https://internetcomputer.org/docs/current/references/bitcoin-how-it-works).
+
+### `bitcoin_get_utxos`
+
+```
```

Comment:
NIT: can you use motoko or candid or something to add syntax highlighting?

Reply:
I played with a few options (rust, js, json) and rust syntax highlighting looks better than others. But of course it's not a perfect solution since it's not candid. Feel free to remove all this annotations.

Reply:
Thanks for playing with all of these options!   :slightly_smiling_face: 

Looking at your branch, I don't think there is a clear winner.
I think `ts` looks pretty good but it only colors the first endpoint under `service bitcoin` for some reason...